docs/index.rst

```diff
@@ -135,6 +137,9 @@ contacting support.
 
 .. [4] only in Python 3.7 and above.
 
+.. [5] While ``asyncio`` is automatically instrumentated, the library uses the
```

Comment:
Maybe better to just point to the asyncio docstring instead of repeating it here 🙂